Formula/azure-cli.rb

```diff
@@ -256,6 +266,11 @@ class AzureCli < Formula
     sha256 "d04bb2425086c3fe86f7bc48915290b13e798497839fbb18ab7f6dffcf98cc3a"
   end
 
+  resource "futures" do
```

Comment:
did you intend to add this?

Reply:
It's ignored lower below with `deps = resources.map(&:name).to_set - ["futures"]` and `pip install azure-cli homebrew-pypi-poet; poet -r azure-cli` includes it so I didn't remove it.

However, it would make sense to remove...

Reply:
>It's ignored lower below with deps = resources.map(&:name).to_set - ["futures"]

Yep, that's why asked :)

>However, it would make sense to remove...

I'm happy either way.